README.md

```diff
@@ -91,6 +91,21 @@ import { reset } from 'ember-window-mock';
 
 This function should be called before all tests that depend on the window mock, preferably in the `beforeEach` hook. See below for some examples!
 
+### Resetting the state in with QUnit 2.0
+
+If your test are using the QUnit 2.0 test syntax, introduced in [RFC 0232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md),
+then you can setup window mock by calling the `setupWindowMock` method:
+
+```js
+import setupWindowMock from 'ember-window-mock/setup-window-mock';
```

Comment:
I think I would prefer a named export directly from `'ember-window-mock'` as the public export:

```
import { setupWindowMock } from 'ember-window-mock';
```

You can leave the file as it is, just reexport the function from `index.js`, similar to the existing `reset` function.

Reply:
The PR now reexports the setupWindowMock method